src/cirrus/lib2/events.py

```diff
@@ -0,0 +1,88 @@
+import json
+import os
+from logging import Logger, getLogger
+from typing import Any, Dict, Tuple
+
+import boto3
+
+from .statedb import StateDB
+from .utils import SNSPublisher
+
+logger = getLogger(__name__)
+
+
+class WorkflowEventManager:
+    """A class for managing payload state change events, inclusing storage of
```

Comment:
```suggestion
    """A class for managing payload state change events, including storage of
```

Reply:
It looks like timestream events are still handled via `StateDB`. I was envisioning the  eventdb stuff would get pulled out of `StateDB` and handled here. Is that still the plan?

Reply:
Yes.  Still the plan.  I am going to get everything to the `"use the WorkflowEventManager"` level using the `StateDB` as it is, and use #235 to address how timestream is handled. 

And thanks for the emacsfingers typo catch.